lib/adagrams.rb

```diff
@@ -0,0 +1,80 @@
+require 'pry'
+
+
+def draw_letters
+  letters_in_hand = ARRAY_OF_LETTERS.shuffle.slice(1..10)
+  return letters_in_hand
+end
+
+ARRAY_OF_LETTERS = ["A", "A", "A", "A", "A", "A", "A", "A", "A", "B", "B", "C", "C", "D", "D", "D", "D", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "F", "F", "G", "G", "G", "H", "H", "I", "I", "I", "I", "I", "I", "I", "I", "I", "J", "K", "L", "L", "L", "L", "M", "M", "N", "N", "N", "N", "N", "N", "O", "O", "O", "O", "O", "O", "O", "O", "P", "P", "Q", "R", "R", "R", "R", "R", "R", "S", "S", "S", "S", "T", "T", "T", "T", "T", "T", "U", "U", "U", "U", "V", "V", "W", "W", "X", "Y", "Y", "Z"]
```

Comment:
It's advisable to store the frequency information in a hash, and then use that hash to create an array. Also, it's best to put this global at the top of your script.


```ruby
letter_counts = {
  "A"=>9, "B"=>2, ...
}
letters = []
letter_counts.each do |letter, count|
  count.times do
    letters << letter
  end
end
```